src/dpp/cluster/guild.cpp

```diff
@@ -23,6 +23,20 @@
 
 namespace dpp {
 
+void cluster::guild_current_member_edit(snowflake guild_id, const std::string &nickname, command_completion_event_t callback) {
+	std::string o;
+	if (nickname.empty()) {
+		o = "{\"nick\": null}";
```

Comment:
you can't use null here.
you must use the nlohmann json null type instead, this will crash.